src/Course/List.hs

```diff
@@ -239,11 +210,18 @@ flattenAgain =
 --
 -- >>> seqOptional (Empty :. map Full infinity)
 -- Empty
-seqOptional ::
-  List (Optional a)
-  -> Optional (List a)
-seqOptional =
-  error "todo: Course.List#seqOptional"
+seqOptional :: List (Optional a) -> Optional (List a)
```

Comment:
take a look at the comment I gave qing for this one. https://github.com/wiserfirst/fp-course/pull/1/commits/8ed0fe1d2aeef6cd5ed0d0dd451e384d1c2a6915#diff-c6476df845e9f5106ab31b5b2abe6647R242

Reply:
oh Interesting, I'll give it a thought!